libdnf.spec

```diff
@@ -12,6 +12,13 @@
 %bcond_without python3
 %endif
 
+%if 0%{?rhel} > 7
```

Comment:
This should be disabled for Fedora 30+ too.

Reply:
Good point if we take removal of python2 seriously. But for sure it will make troubles.

Reply:
Make it a Fedora 30 proposed change, along with the mass Python 2 removal stuff.